tests/integration/suite/daprd/conversation/http/basic.go

```diff
@@ -0,0 +1,83 @@
+/*
+Copyright 2024 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implieh.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package http
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"net/http"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/dapr/dapr/tests/integration/framework"
+	"github.com/dapr/dapr/tests/integration/framework/client"
+	procdaprd "github.com/dapr/dapr/tests/integration/framework/process/daprd"
+	"github.com/dapr/dapr/tests/integration/suite"
+)
+
+func init() {
+	suite.Register(new(basic))
+}
+
+type basic struct {
+	daprd *procdaprd.Daprd
+}
+
+func (b *basic) Setup(t *testing.T) []framework.Option {
+	b.daprd = procdaprd.New(t, procdaprd.WithResourceFiles(`
+apiVersion: dapr.io/v1alpha1
+kind: Component
+metadata:
+  name: conversation
+spec:
+  type: conversation.echo
+  version: v1
+  metadata:
+  - name: model
+    value: gpt4o
+  - name: key
+	value: testkey`))
+
+	return []framework.Option{
+		framework.WithProcesses(b.daprd),
+	}
+}
+
+func (b *basic) Run(t *testing.T, ctx context.Context) {
+	b.daprd.WaitUntilRunning(t, ctx)
+
+	postURL := fmt.Sprintf("http://localhost:%d/v1.0/conversation/echo", b.daprd.HTTPPort())
```

Comment:
How is this working if the API is `v1.0-alpha1`? https://github.com/dapr/dapr/pull/7858/files#diff-982843874043a320e810f10e11e97448e5e07c3841388262dcda246d04590ddbR30

Reply:
Ok, just waiting on the test. Otherwise LGTM